gatsby-config.js

```diff
@@ -124,13 +119,22 @@ const manifestPlugin = {
 /* eslint-enable @typescript-eslint/camelcase */
 
 let plugins = _.compact([
-  'gatsby-plugin-typescript',
+  {
+    resolve: 'gatsby-plugin-typescript',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)